src/api/mapping.ts

```diff
@@ -0,0 +1,21 @@
+import express from 'express'
+import uuid4 from 'uuid/v4'
+import { setData, getData, getMappings } from './lib/data'
+import { Mapping } from './types/general' 
+const mappingRouter = express.Router()
+
+mappingRouter.post('/', (req, res) =>{
+  const domainKeys = getMappings()
+  const mappingObject: Mapping = {
+    'domain': req.body.domain,
+    'port' : req.body.port,
+    'ip' : req.body.ip,
```

Comment:
this should be defaulted to `127.0.0.1`, which is `localhost`. so... `'ip': req.body.ip || '127.0.0.1'`